next-app/src/lib/fetch/server.ts

```diff
@@ -11,57 +11,68 @@ import { throwError } from '$lib/error'
  * @param { RPC } rpc
  */
 export async function sf(rpc) {
-	const { name, args = [], cookie } = rpc
+  const { name, args = [], cookie } = rpc;
 
-	const body = {
-		id: Date.now(),
-		method: name,
-		params: {
-			orderedParams: args,
-		},
-	}
+  const body = {
+    id: Date.now(),
+    method: name,
+    params: {
+      orderedParams: args,
+    },
+  };
 
-	const results = await customFetch(`${process.env.API_HOST}/api/sf`, 'post', body, cookie)
+  const results = await customFetch(
```

Comment:
maybe we can widen this conversion a little more... I don't like this result personally.

Reply:
The long description about prettier's `printWdith` option confused me, but playing with it a bit: if we bump it up to 100 or 120 (or 130 - that's actually the max-width I'm used to) this change gets reverted.

I'd suggest we go with 120, giving the develop more freedom here.

Reply:
Actually, "more freedom" is incorrect, because prettier will still always do what it wants, it'll just do something different if we give it more space.

Reply:
I like the possibility of going to a larger max-width, the wider the better for me personally... I prefer the ability to put things on a different line if/when it makes sense (to me)